Source/Blazorise/Components/Breadcrumb/BreadcrumbItem.razor.cs

```diff
@@ -97,7 +97,7 @@ internal void NotifyRelativeUriChanged( string relativeUri )
     #region Properties
 
     /// <summary>
-    /// Gets or sets the item active state.
+    /// If set to true, renders <c>span</c> instead of <c>a</c> element.
```

Comment:
This comment is incorrect. In the active state, it only means the item or a link will have some active class and can look unclickable.

Reply:
I copied that from the current docs.